### ISSUE TITLE
Vurdere å innføre bruk av SKAL, KAN, BØR i versjon 2.0

### ISSUE THREAD

**Issue by rockphotog** [HL7Norway/best-practice#90]
Title says it all. Lukker også issue #40 

RFC som HL7 har pekt på: https://www.ietf.org/rfc/rfc2119.txt
(ref https://www.hl7.org/FHIR/2015May/conformance-rules.html)

**Comment by kennethmyhra** [comment]
Beskrev noe lignende i forbindelse med HelseAPI implementasjonsguiden https://helsenorge.atlassian.net/wiki/spaces/HELSENORGE/pages/142835738/Generell+veiledning

Pekte hovedsaklig på bruken av det i FHIR spesifikasjonen

**Comment by thomiz** [comment]
Ja, dette må i såfall innføres i IG'ene våre og metoden for områdeprofiler. Akkurat det er jo en jobb. Men hvis vi er enige om at dette er en god ide, så kan vi ta innføringen av den konkrete ordlyden etterhvert som dokumentene revideres. Det bør også noen linjer om dette inn i metoden og informasjon om at IG'er bør bruke et slikt språk bør nok også nevnes i den gode implementasjonsguide.

**Comment by rockphotog** [comment]
ISO: https://www.iso.org/foreword-supplementary-information.html

> "shall" indicates a requirement
> "should" indicates a recommendation
> "may" is used to indicate that something is permitted
> "can" is used to indicate that something is possible, for example, that an organization or individual is able to do something

[Helsedirektoratets](https://www.helsedirektoratet.no/om-oss/om-helsedirektoratets-normerende-produkter) generelle nivåer gjelder hele produkter og passer ikke enkelt-krav. 

FHIR baserer seg på RFC 2119, men med en tvist. Usikker på hva HL7 International sin eventuelle annen praksis er.

https://www.hl7.org/fhir/conformance-rules.html

https://www.rfc-editor.org/rfc/rfc2119.txt

EDIT: Ser jeg repeterer meg selv fra starten - uansett så mener jeg dette fortsatt er aktuelt, spesielt siden metoden for områdeprofil ikke ligger ute som anbefalt fra Helsedirektoratet: Det gjør det enklere å revidere den.
